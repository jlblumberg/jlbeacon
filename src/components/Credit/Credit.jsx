import React from 'react';
import { Typography, Box, Link } from '@material-ui/core';

const Credit = () => {

  return(
    <Box pb={3} pt={3} id='credit'>
      <Typography variant="body2" color="textSecondary" align="center">
        {'Built by Josh Blumberg to try to secure an interview at Beacon. View the source code on '}
        <Link color="inherit" href="https://github.com/jlblumberg/jlbeacon">
          GitHub 
        </Link>
        {'.'}
      </Typography>
    </Box>
  );

}

export default Credit;
