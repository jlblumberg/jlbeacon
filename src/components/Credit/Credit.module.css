@media (max-width: 770px) {

  .credit {
    max-width: 80%;
    margin: 5%;
  }

}