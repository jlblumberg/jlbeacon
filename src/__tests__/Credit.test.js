import React from 'react';
import { shallow } from 'enzyme'
import Credit from '../components/Credit/Credit';

describe('Credit', () => {

  it('renders correctly', () => {
    let wrapper = shallow(<Credit />);
    expect(wrapper).toMatchSnapshot();
  });

  it('has a credit line at the bottom', () => {
    const wrapper = shallow(<Credit />);
    expect(wrapper.find('#credit').text()).toContain('Josh Blumberg');
  });

});
