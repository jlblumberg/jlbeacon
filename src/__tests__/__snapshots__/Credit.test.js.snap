// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credit renders correctly 1`] = `ShallowWrapper {}`;
